hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManager.java

```diff
@@ -180,7 +180,7 @@ ContainerInfo getMatchingContainer(long size, String owner,
    * @return ContainerInfo for the matching container.
    */
   ContainerInfo getMatchingContainer(long size, String owner,
-      Pipeline pipeline, List<ContainerID> excludedContainerIDS);
+      Pipeline pipeline, Set<ContainerID> excludedContainerIDS);
```

Comment:
Can we change here to Collection<ContainerID> to make the interface neutral to implementation? The underlying can be List or Set.